tests/unit/services/user.test.js

```diff
@@ -0,0 +1,239 @@
+                {
+                    id: '6f7a7188-5b7d-4dbb-8a43-12031b94971e',
+                    login: 'John Doe',
+                    password: 'secret',
+                    age: 21,
+                    isDeleted: false
+                },
+                {
+                    id: '7e4e5a21-42c2-46a8-add1-ea799f547d5f',
+                    login: 'Jane Doe',
+                    password: 'secret',
+                    age: 25,
+                    isDeleted: false
+                },
+                {
+                    id: '1ab97ffa-bbef-4619-89d6-7aa218444353',
+                    login: 'John Carmack',
+                    password: 'secret',
+                    age: 55,
+                    isDeleted: false
+                }
+            ]);
+        });
+
+        const output = await UserService.findAll();
+
+        expect(output).toEqual(expect.arrayContaining(
+            [
+                expect.objectContaining({
+                    id: '6f7a7188-5b7d-4dbb-8a43-12031b94971e',
+                    login: 'John Doe',
+                    password: 'secret',
+                    age: 21,
+                    isDeleted: false
+                })
+            ])
+        );
+    });
+
+    test('It should return individual user', async () => {
+        // Mock Model findOne
+        jest.spyOn(User, 'findOne').mockImplementation(() => {
+            return Promise.resolve(
+                {
+                    id: '6f7a7188-5b7d-4dbb-8a43-12031b94971e',
+                    login: 'John Doe',
+                    password: 'secret',
+                    age: 21,
+                    isDeleted: false
+                }
+            );
+        });
+
+        const output = await UserService.findById('6f7a7188-5b7d-4dbb-8a43-12031b94971e');
+
+        expect(output).toEqual(
+            expect.objectContaining({
+                id: '6f7a7188-5b7d-4dbb-8a43-12031b94971e',
+                login: 'John Doe',
+                password: 'secret',
+                age: 21,
+                isDeleted: false
+            })
+        );
+    });
+
+    test('It should return exception if user not found', async () => {
+        // Mock Model findOne
+        jest.spyOn(User, 'findOne').mockImplementation(() => {
+            return Promise.resolve({
+                'name': 'UserNotFoundError',
+                'success': false,
+                'status_code': 404,
+                'errors': {
+                    'message': 'User Not Found'
+                }
+            });
+        });
+
+        const output = await UserService.findById('foo');
+
+        expect(output).toEqual(
+            expect.objectContaining({
+                'name': 'UserNotFoundError',
+                'success': false,
+                'status_code': 404,
+                'errors': {
+                    'message': 'User Not Found'
+                }
+            })
+        );
+    });
+
+    test('It can suggest usernames', async () => {
```

Comment:
Also, try using `test.each` and `describe.each` for tests with repeated structure